lib/util/trip/trip.dart

```diff
@@ -4,6 +4,8 @@ import '../model.dart';
 import '../search/position.dart';
 
 abstract class Trip extends Model {
+  static const int maxSelectableSeats = 8;
```

Comment:
why did we reduce that to 8?

Reply:
i'll change it in the database so it's only >= 1

Reply:
Oh okay then that should be changed in this PR as well.

Although that makes the display more difficult for us... The 100 seat icons in a row would overflow completely.

Reply:
I think we should place some sort of limit on the number of seats. If your cat taps on the + icon 100 times, you have to click on the - icon 100 times or I have to implement a reset functionality. I think the increment field is not really designed to handle even high 2-digit numbers and at some point we should safeguard against silly inputs as well. What Fynn found isn't a car anymore, it's under the tab "trucks and vans" on the website, at which point we have to decide whether we want to accomodate 49-seater buses as well. BlaBlaCar's limit is 4, by the way!

Reply:
then let's just leave 8?

Reply:
Yes, can you add it in supabase again?